src/snapred/backend/recipe/algorithm/LoadGroupingDefinition.py

```diff
@@ -92,10 +92,6 @@ def validateInputs(self) -> Dict[str, str]:
         if groupingFileExt not in self.all_extensions:
             errors["GroupingFilename"] = f"Grouping file extension {groupingFileExt} not supported"
 
-        if groupingFileExt not in self.supported_xml_file_extensions:
-            if not self.getProperty("InstrumentDonor").isDefault:
-                logger.warning("InstrumentDonor will only be used if GroupingFilename is in XML format.")
-
```

Comment:
Do we want to remove logging the warning, or displaying the warning in a pop-up box?

Reply:
We could potentially change this to use the mantid logger to log the warning, and thus still get a warning and not display a popup.